.travis.yml

```diff
@@ -12,19 +14,34 @@ script:
 
 jobs:
   include:
-    - python: '3.6'
-      env: TOXENV=docs
-
-    - python: '3.7'
-      env: TOXENV=py37
-
-    - python: '3.6'
-      env: TOXENV=py36
-
-    - python: '3.7'
-      env: TOXENV=linting
-
+  - python: '3.6'
+    env: TOXENV=docs
+
+  - python: '3.7'
+    env: TOXENV=py37
+
+  - python: '3.6'
+    env: TOXENV=py36
+
+  - python: '3.7'
+    env: TOXENV=linting
+
+  - stage: deploy
+    python: '3.6'
+    env:
+    install: pip install -U setuptools setuptools_scm
+    script: skip
+    deploy:
+      provider: pypi
+      distributions: sdist bdist_wheel
+      user: loliveira
```

Comment:
Use the new token feature from PyPI: https://pypi.org/help/#apitoken

Reply:
Done